packages/expo-modules-core/android/src/main/java/expo/modules/interfaces/taskManager/TaskServiceProviderHelper.kt

```diff
@@ -0,0 +1,36 @@
+package expo.modules.interfaces.taskManager
+
+import android.content.Context
+import expo.modules.core.ModulePriorities
+import expo.modules.core.interfaces.Package
+
+class TaskServiceProviderHelper {
```

Comment:
just a stylistic comment - `class ... companion object` can be written as just `object` in Kotlin AFAIK. Not sure how it interops with java though.

Reply:
Done! Thanks :)